.vscode/tasks.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Was this file added by accident?

Reply:
I agree ^.

Reply:
I added it intentionally, but I think you're right in this case.  Will remove.

Reply:
And also will add a `.gitignore`